server/task/task-memory/src/main/scala/org/apache/james/task/eventsourcing/DecisionProjection.scala

```diff
@@ -35,6 +35,13 @@ case class DecisionProjection(status: Option[Status], latestUpdateAdditionalInfo
         case event: AdditionalInformationUpdated => DecisionProjection(status, Some(event.additionalInformation.timestamp))
       }
   }
+
+  val currentStatus: Option[Status] = status
+
+  val latestAdditionalInformationTimestamp: Option[Instant] = latestUpdateAdditionalInformationUpdate
+
+  def isNewerThanLatestAdditionalInformation(timestamp: Instant) : Boolean = latestAdditionalInformationTimestamp.forall(timestamp.isAfter)
```

Comment:
I don't understand what is supposed to be newer.
What about: `additionalInformationOlderThan(timestamp)`